packages/components/src/TextField/TextField.test.tsx

```diff
@@ -129,4 +129,52 @@ describe('TextField', () => {
       expect(inputElement.value).toHaveLength(MAX_LENGTH);
     });
   });
+
+  it('При переводе фокуса с инпута пробел в конце текста удаляется', async () => {
```

Comment:
```suggestion
  it('Пробел в конце текста удаляется при переводе фокуса с инпута', async () => {
```

https://kaluga-astral.github.io/style-guide/docs/rules/tests/structure#%D1%81%D1%82%D1%80%D1%83%D0%BA%D1%82%D1%83%D1%80%D0%B0-%D1%82%D0%B5%D1%81%D1%82%D0%B0-%D0%B4%D0%BE%D0%BB%D0%B6%D0%BD%D1%8B-%D1%81%D0%BE%D0%BE%D1%82%D0%B2%D0%B5%D1%82%D1%81%D1%82%D0%B2%D0%BE%D0%B2%D0%B0%D1%82%D1%8C-aaa-arrange-act-assert